docs/source/generate_agent_list.py

```diff
@@ -3,10 +3,83 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.abs
 
+import parlai.core.agents
 import parlai.agents
+from parlai.core.params import ParlaiParser
 
 import os
 import pkgutil
+import importlib
+import inspect
+import io
+import csv
+
+
+"""
+Extract the readme and CLI args for all of the standard agents.
+"""
+
+
+def _make_argparse_table(class_):
+    """
+    Build the reStructuredText table containing the args and descriptions.
```

Comment:
are we inserting an RST table into an markdown file? does that work?

Reply:
uhhhhh yesss. lol

Reply:
I tried to build it in markdown but the markdown plugin doesn't support tables